twake/backend/core/src/Twake/Discussion/Entity/Message.php

```diff
@@ -498,6 +596,7 @@ public function getAsArray()
             "content" => $this->getContent(),
             "user_specific_content" => $this->getUserSpecificContent(),
             "increment_at_time" => $this->getIncrementAtTime(),
+            "api_object" => $api_object,
```

Comment:
You need to mix the content of "api_object" with the main array, with something like:
return array_merge(Array(..), $api_object)

Reply:
Done 👍🏻